course_info/static/course_info/js/controllers/PeopleController.js

```diff
@@ -72,6 +72,27 @@
             return null;
         };
 
+        $scope.getSchool = function() {
+            return $scope.courseInstance.school
+        }
+
+        $scope.allowDualEnrollment = function(member, roleId) {
+            // Check if enrollment request is for TA/Student roles in GSD.
+            // For this school only, we allow a given HUID to be enrolled
+            // twice if (and only if) they are enrolled as one of the
+            // two eligible roles (TA/Student).
+
+            var eligibleSchool = 'GSD'
+            var eligibleRoleIds = [0, 5]
+            var memberRole = member[0].role.role_id
+
+            return ($scope.getSchool() === eligibleSchool // ensure course is in GSD
```

Comment:
Would the getSchool always be uppercase GSD? If so it's fine, otherwise making this check case insesnsitive maybe a good idea

Reply:
Added `toLowerCase()` to avoid case issues.